src/js/components/Select/Select.js

```diff
@@ -7,12 +7,16 @@ import { DropButton } from '../DropButton';
 import { Keyboard } from '../Keyboard';
 import { TextInput } from '../TextInput';
 import { withForwardRef, withTheme } from '../hocs';
-import { colorIsDark, evalStyle } from '../../utils';
+import { borderStyle, colorIsDark, evalStyle } from '../../utils';
 
 import SelectContainer from './SelectContainer';
 import doc from './doc';
 
 const SelectTextInput = styled(TextInput)`cursor: pointer;`;
+const StyledSelectBox = styled(Box)`
+  ${props => !props.plain && borderStyle};
+  ${props => props.theme.select && props.theme.select.extend}
```

Comment:
I wonder if this should be `props.theme.select.control.extend`? That would allow for `props.theme.select.drop.extend`.

Reply:
yup